src/CSI.IBTA.AuthService/Authentication/PasswordHasher.cs

```diff
@@ -0,0 +1,52 @@
+﻿using CSI.IBTA.AuthService.Interfaces;
+using System.Security.Cryptography;
+
+namespace CSI.IBTA.AuthService.Authentication
+{
+    public class PasswordHasher : IPasswordHasher
```

Comment:
```suggestion
    internal class PasswordHasher : IPasswordHasher
```

I will not replicate this request later, but classes registered with DI preferably should have stricter access modifier when possible.